CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.4.0
```

Comment:
We have already released 1.4.0 on GitHub.
It'd be better to use 1.5.0 for these changes.